replication/README.md

```diff
@@ -366,6 +366,15 @@ message GetVolumeReplicationInfoResponse {
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // Holds the last sync duration.
+  // This duration is in seconds.
+  // This field is REQUIRED.
+  int64 last_sync_duration = 2;
```

Comment:
a couple of questions

What is the difference between last_sync_time and last_sync_duration?
Why is this int64 instead of `.google.protobuf.Timestamp`?
Why is this a required field instead of optional one?

Reply:
last_sync_time represents the exact date and time when it was last synced, which last_sync_duration mentions the time used to excute the last sync which is in seconds, which answers the second query and yeah it need not be required. We can keep it optional so that other replication system which donot have these info will not report it. I will update the PR with more details. 
@Madhu-1

Reply:
It is already in seconds, it's a duration, right? we don't need to have a timestamp. We can directly store it in int64. That should be fine. please correct me if I am wrong.

Reply:
if it's a duration is it possible to use https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto proto here?

Reply:
ceph-csi will have it in int64 itself, so if this is already in the same type we will save the conversion at both the end. Hence, would prefer to have it in int64.

Reply:
Just so you know, this spec change is not specific to cephcsi. It should be a generic spec change to accommodate different storage providers.

Reply:
yeah get it, cool will update the pr.

Reply:
@Madhu-1 done, please do check.